src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Behaviors/UserStoppedTypingBehavior_Tests.cs

```diff
@@ -20,13 +20,13 @@ public async Task ShouldExecuteCommandWhenTimeThresholdHasExpired()
 
 			// act
 			entry.Text = "1";
-			await Task.Delay(defaultTimeThreshold + 100);
+			await Task.Delay(defaultTimeThreshold * 2);
```

Comment:
This commit popped out in my notifications, so quick question:

- A 100ms difference should be granular enough to trigger the command. If not: is this wide range in possible execute timing an issue?
- Since we're using the same offset in every test, why not introduce a second parameter: testTimeTreshold?

Reply:
In a perfect world, yes an extra 100ms is enough time. However [`Task.Delay` isn’t a perfect Timer](https://stackoverflow.com/a/31126576/5953643), and it was causing these tests to fail randomly. Increasing the timeout guarantees we won’t get any false failures.